README.md

```
@@ -1,62 +1,65 @@
+### Client Initialization
 
-*   `GC_BASE_URL`: The base URL of your Groundcover API endpoint.
-*   `GC_API_KEY`: Your Groundcover API key.
-*   `GC_BACKEND_ID`: Your Groundcover Backend ID.
+The SDK client requires a configured transport stack that handles authentication, retries, and other custom behaviors. Here's an example of how to set up the client:
 
 ```go
 package main
 
 import (
 	"context"
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)